yt/yt/server/lib/misc/job_reporter.cpp

```diff
@@ -326,7 +367,13 @@ class TJobProfileRowlet
 
     size_t EstimateSize() const override
     {
-        return Report_.EstimateSize();
+        return ::NYT::EstimateSizes(
+            Report_.OperationId().Underlying(),
+            Report_.JobId().Underlying(),
+            Report_.Profile().value_or(NJobAgent::TJobProfile{}).Type,
+            /*partIndex*/ int{0},
+            Report_.Profile().value_or(NJobAgent::TJobProfile{}).Blob,
+            Report_.Profile().value_or(NJobAgent::TJobProfile{}).ProfilingProbability);
```

Comment:
Maybe we can implement `EstimateSize` for `NJobAgent::TJobProfile`?

Reply:
When I was writing the code I thought `estimate_size_helpers` are in regular generic misc, but now I see that they are actually in `server/lib/misc`, same as the rest of the code. 

I do not have a strong opinion, I guess we can dedicate a section to specific types.

Reply:
I believe we can define an `EstimateSize` specialization for `TJobProfile` next to the definition of `TJobProfile`?